packages/ember-css-modules/package.json

```diff
@@ -72,7 +72,7 @@
     "broccoli-bridge": "^1.0.0",
     "broccoli-concat": "^3.2.2",
     "broccoli-css-modules": "^0.7.0 || ^0.8.0",
-    "broccoli-funnel": "^2.0.1",
+    "broccoli-funnel": "^3.0.7",
```

Comment:
Will need to ponder a bit what this means for semver, since ECM hasn't cut a major since Node 6 was still supported. Pragmatically Node 6 and 8 are both more than a year out of maintenance support themselves and I _hope_ no one is using them, but letter of the law dropping support for them should probably mean a major bump here, which would be a bit of a pain.

Reply:
Ya, sorry about that. I _might_ be able to backport these changes to the 2.x branch of broccoli-funnel, but I don't even know if our CI works that far back at this point.